### PR TITLE
Move Databricks variables into blossom-lib [databricks]

### PR DESCRIPTION
**Pull request by NvTimLiu**
Move Databricks variables into blossom-lib to share the variables among Jenkins files.

Keep only one copy of the Databricks variables in blossom-lib to make it easy to maintain.

Signed-off-by: Tim Liu <timl@nvidia.com>

### CODE REVIEW

**Comment by NvTimLiu** [comment]
build

**Comment by pxLi** [comment]
can you move this directly targeting to 23.04? we should not merge non-bug fix CI update to 23.02 any more, thanks~

File: jenkins/Jenkinsfile-blossom.premerge [hunk 1]
```diff
@@ -488,7 +422,18 @@ void databricksBuild() {
                     sh "tar -zcf spark-rapids-ci.tgz *"
                     def CREATE_PARAMS = " -r $DATABRICKS_RUNTIME -w $DATABRICKS_HOST -t $DATABRICKS_TOKEN" +
                             " -s $DB_TYPE -n CI-${BUILD_TAG}-${BASE_SPARK_VERSION} -k \"$DATABRICKS_PUBKEY\" -i $IDLE_TIMEOUT" +
-                            " -d $DATABRICKS_DRIVER -o $DATABRICKS_WORKER -e $NUM_WORKERS -f $INIT_SCRIPTS"
+                            " -d $DATABRICKS_DRIVER -o $DATABRICKS_WORKER -e $NUM_WORKERS"
+
+                    // handle init scripts if exist
+                    if ( env.INIT_SCRIPTS ){
```
Comment (pxLi) [inline jenkins/Jenkinsfile-blossom.premerge:428]: nit: format. if intellij could simply use `Reformat Code`

Comment (NvTimLiu) [reply jenkins/Jenkinsfile-blossom.premerge:428]: fixed

**Comment by NvTimLiu** [comment]
build

**Review by tgravescs — COMMENTED** [review]
doesn't looks like it affects how the developer would build so looks fine to me, liike Peixin said can we move to 23.04?

**Review by revans2 — DISMISSED** [review]
Although I am no expert on this code.

**Comment by NvTimLiu** [comment]
retarget to23.04

**Comment by NvTimLiu** [comment]
build

**Comment by pxLi** [comment]
currently databricks ENV is suffering from broken python dependencies again. try CI later
```

[2023-02-10T01:18:07.585Z] Traceback (most recent call last):

[2023-02-10T01:18:07.585Z]   File "/home/ubuntu/spark-rapids/integration_tests/runtests.py", line 17, in <module>

[2023-02-10T01:18:07.585Z]     from pytest import main

[2023-02-10T01:18:07.585Z] ModuleNotFoundError: No module named 'pytest'
```

**Review by tgravescs — APPROVED** [review]

**Comment by NvTimLiu** [comment]
build

**Comment by NvTimLiu** [comment]
build

**Pull request merged**